livekit-agents/livekit/agents/voice/task_activity.py

```diff
@@ -940,7 +940,7 @@ async def _read_messages(
 
                 self._rt_session.interrupt()
                 try:
-                    await self._rt_session.generate_reply()
+                    await self.generate_reply()
```

Comment:
makes a ton of sense, can u add a comment, so we avoid to do the same mistake in the future